src/connectors/range/connectRange.js

```diff
@@ -226,20 +229,27 @@ export default function connectRange(renderFn, unmountFn = noop) {
         const currentRange = this._getCurrentRange(stats);
         const start = this._getCurrentRefinement(helper);
 
-        renderFn(
-          {
-            refine: this._refine(helper, currentRange),
-            format: rangeFormatter,
-            range: currentRange,
-            widgetParams: {
-              ...widgetParams,
-              precision,
-            },
-            start,
-            instantSearchInstance,
+        let refine;
+
+        if (!results) {
+          // On first render pass an empty range
+          // to be able to bypass the validation
+          // related to it
+          refine = this._refine(helper, {});
```

Comment:
don't we have access to the current refinement already? as far as I can tell the validation gets done only once you call refine, and that would be the same issue as empty object, right? Or do I misunderstand it?

Reply:
I have no idea here, I just took back the existing logic.